WireUI/Sources/WireMainNavigationUI/Protocols/Sidebar/MainSidebarMenuItemRepresentable.swift

```diff
@@ -18,19 +18,8 @@
 
```

Comment:
I think changes in this file are no longer necessary. I will revert them.

Reply:
Deleting it is fine.